stripe/test/test_util.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import print_function
+
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+from stripe import util
+from stripe.test.helper import StripeUnitTestCase
+
+import __builtin__
+
+PRINT_FUNC_STRING = __builtin__.__name__ + '.print'
+
+
+class DashboardLinkTests(StripeUnitTestCase):
```

Comment:
This is super nitpicky, but I notice that most of the other test suites are kind of modeled after a single class like `UtilTests`. Do you think that we maybe should consider following that pattern here too? I see that you're using an inheritance model here for some of the suites, but I wonder if that could just as easily be refactored to a set of test helpers.

And thanks for getting some util tests started BTW! Crazy that we didn't have any before.